website/docs/guides/docs/docs-multi-instance.mdx

```diff
@@ -139,6 +139,22 @@ We consider that the `product` instance is the most important one, and make it t
 
 :::
 
+:::note
+
+Additional docs instances can't share the docs/ directory, e.g.:
+
+docs/
+docs/product
+docs/community
+
+Instead, each instance holds their own directory, e.g.:
+
+docs/
+docs-product/
+docs-community/
+
+:::
```

Comment:
that's not totally true

The thing you want to avoid is nesting docs folder instances.

You can have 2 instances with 2 paths:
- `path: 'docs/product',`
- `path: 'docs/community',`

Also your example show 3 instances, not very consistent with examples above. Users are generally confused when we do this.

Reply:
@slorber Thanks for your remarks! Good to know.